libraries/botbuilder-ai/src/luisRecognizer.ts

```diff
@@ -344,11 +344,17 @@ export class LuisRecognizer implements LuisRecognizerTelemetryClient {
     }
 
     // Gets a value indicating whether determines whether to log personal information that came from the user.
+    /**
+     *@returns True if will log personal information into the IBotTelemetryClient.TrackEvent method; otherwise the properties will be filtered.
```

Comment:
Does IBotTelemetryClient exist in JS or is this a copy of a .NET comment??

Reply:
It doesn't, the one that exists is without the 'I', you can use this change as a reference for telemetry related things.
https://github.com/southworks/botbuilder-js/pull/298/files#diff-0989e63acc2287bad60667499884de5615383f50dd2360e80128a48035a77728R112